opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/__init__.py

```diff
@@ -32,20 +33,32 @@ def __init__(
         labels: Tuple[Tuple[str, str]],
         aggregator: Aggregator,
         resource: Resource,
+        process_start: int,
```

Comment:
How come we need so much information in the `MetricRecord`? This was simply used to encapsulate what we needed to export. Can't these values be derived from the aggregator?